cmake/tmxparserConfig.cmake.in

```diff
@@ -1 +1,6 @@
-include("${CMAKE_CURRENT_LIST_DIR}/tmxparserTargets.cmake")
\ No newline at end of file
+include(CMakeFindDependencyMacro)
+
+find_dependency(tinyxml2 REQUIRED)
+find_dependency(ZLIB REQUIRED)
```

Comment:
We need `CONFIG` here to not to fall into system version:
- https://docs.hunter.sh/en/latest/packages/pkg/ZLIB.html